scripts/tests/twister/robotframework_integration/test_harness_robotframework.py

```diff
@@ -0,0 +1,240 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
If this is not an exact copy of an already existing file, the copyright should be 2024 and Carl Zeiss Meditec AG